src/components/shared/sidebar/sidebar.tsx

```diff
@@ -62,6 +66,9 @@ function Sidebar(props: any) {
       code === 'dark' ? setTheme('dark') : setTheme('light');
       props.setToggleDis(false);
     }
+    if (title === 'About Map of Pi' && code === 'version') {
```

Comment:
will this get along with internationalization?

Reply:
Great point!  
There will be a separate task to internationalize the contents.